.github/workflows/main.yml

```diff
@@ -0,0 +1,21 @@
+name: All the CI
+
+on: [push, pull_request]
+
+jobs:
+  unit:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Unit Tests
```

Comment:
Is this well aligned?

Reply:
Seems so, CI runs properly

Reply:
Yep, the steps in CI are correct 😄

Reply:
My app was showing an extra space. Sorry for that.